src/main/java/brandkon/product/ProductService.java

```diff
@@ -0,0 +1,63 @@
+package brandkon.product;
+
+import brandkon.Brands.BrandRepository;
+import brandkon.Brands.DTO.BrandDetailsResponse;
+import brandkon.product.DTO.ProductDetailed;
+import brandkon.product.DTO.ProductResponse;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    private final BrandRepository brandRepository;
+
+    public ProductService(ProductRepository productRepository, BrandRepository brandRepository) {
+        this.productRepository = productRepository;
+        this.brandRepository = brandRepository;
+    }
+
+    public List<ProductResponse> findBrandAll(Long brandId) {
```

Comment:
```suggestion
    public List<ProductResponse> findAllByBrand(Long brandId) {
```
이런 형식의 이름이 더 나을 것 같아요